.changeset/shiny-falcons-fly.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage/plugin-events-node': minor
```

Comment:
```suggestion
'@backstage/plugin-events-node': patch
```

Think we can keep this as a patch, no breaking changes there

Reply:
I used `minor` due to the required fields added to `RequestDetails`. However, that should only impact tests and no production code for request validation. Therefore, I guess `patch` might be ok as well. 🤔 

I will adjust it. Let me know if that info changes your stand here.